app/migrations/Version20160310181749.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Sylius\Migrations;
```

Comment:
Several migration files don't have this block, so I suppose we can add it to all of them in a separate PR.